src/main/java/games/strategy/internal/persistence/serializable/VersionPersistenceDelegate.java

```diff
@@ -0,0 +1,22 @@
+package games.strategy.internal.persistence.serializable;
+
+import java.io.IOException;
+
+import games.strategy.persistence.serializable.AbstractPersistenceDelegate;
+import games.strategy.util.Version;
+
+/**
+ * A persistence delegate for the {@code Version} class.
+ *
+ * <p>
+ * Instances of this class are immutable.
+ * </p>
+ */
+public final class VersionPersistenceDelegate extends AbstractPersistenceDelegate {
```

Comment:
What are your thoughts on merging the persistence delegate with the proxy? I think that would reduce the number of serialization proxy classes needed to just one.

I can see two ways to do this:
1. bit directly, have the proxy extend `AbstractPersistenceDelegate`
2. Create a base class or interface for the proxy classes and require them to each have a `getPersistenceDelegate()` method. We could make that a bit nicer even if the return value is a functional interface, so in `VersionProxy.java` we could have something like:
```
// instead of replaceObject, exact return type is pseudo code..
@Override
public Supplier<? super ProxyDelegate> getPersistenceDelegate() {
  return obj -> (obj instanceof Version) ? new VersionProxy((Version) obj) : super.replaceObjecxt(obj);
}
```

I suspect with route 2 and some common templated scaffolding code that could be simplified down to (YMMV, the templating might run into a road block somewhere, but seems feasible):
```
public Supplier<PersistenceDelegate<Version>> getPersistenceDelegate() {
  return version -> new VersionProxy(version);
}
```